.github/ISSUE_TEMPLATE/Standard-Feature-Issue-Template.md

```diff
@@ -22,31 +22,44 @@ so that < why do you need this (short description of the business value >.
 ## Notes
 < bulleted list of special notes or suggestions to the development team on the implementation of the changes. >  
 
-## Definition of Ready
+## Ready for Planning
 
 - [ ] The Issue maintains standard format for Issues and meets INVEST
-- [ ] Issue Defined and written appropriately
-- [ ] Acceptance Criteria Defined
-- [ ] User acceptance test cases defined
+- [ ] Story and Business Value complete
+- [ ] Initial Acceptance Criteria Defined
 - [ ] The person who will Approve story identified
-- [ ] Dependencies Identified
+
+## Ready for Ready to Work
+- [ ] Acceptance Criteria Refined
 - [ ] Issue size estimated by Dev Team
-- [ ] Proposed solution accepted
+- [ ] Dependencies Identified
 
-## Definition of Done
+## Ready for Review
+- [ ] Proposed solution defined and accepted
```

Comment:
Accepted by who?

Reply:
Should be the Approver. I will implement that change

Reply:
OK. You're going to have to make sure that's clearly communicated to the team. That's a process change.